support-frontend/assets/pages/digital-subscription-checkout/components/helpers.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+
+const getGiftOrderSummaryText = (giftType: 'Annual' | 'Quarterly', price: string) => ({
+  period: `${giftType === 'Annual' ? 12 : 3} month gift subscription`,
+  cost: `You'll pay ${price} for ${giftType === 'Annual' ? 12 : 3} months`,
+});
```

Comment:
```suggestion

import { Annual, Quarterly } from 'helpers/billingPeriods';

const getGiftOrderSummaryText = (giftType: typeof Annual | typeof Quarterly, price: string) => {
  const numberOfMonths = giftType === Annual ? 12 : 3;
  return {
    period: `${numberOfMonths} month gift subscription`,
    cost: `You'll pay ${price} for ${numberOfMonths} months`,
  };
};
```

Reply:
It would be even better to create a new type in billingPeriods.js and use that as the type of the `giftType` parameter.
`export type DigitalBillingPeriod = typeof Monthly | typeof Annual | typeof Quarterly;`

Reply:
Very true! Thanks for this suggestion.